conf/openshift/openshift.json

```diff
@@ -526,15 +527,16 @@
       }
     },
     {
-      "kind": "DeploymentConfig",
-      "apiVersion": "v1",
+      "kind": "StatefulSet",
+      "apiVersion": "apps/v1beta1",
```

Comment:
Is this going to change also?

Reply:
yes. solr should be a statefulset so the master/slave replication configuration could work